tasks/main.yml

```diff
@@ -6,37 +6,45 @@
     - curl
     - build-essential
     - libssl-dev
+  become: true
   tags: nvm
 
 - name: Install nvm
-  sudo: yes
-  sudo_user: "{{ nvm.user }}"
+  become: "{{ nvm.user }}"
   git: repo=https://github.com/creationix/nvm.git dest=~/.nvm version={{ nvm.version }}
   tags: nvm
 
 - name: Source nvm in ~/.profile
-  sudo: yes
-  sudo_user: "{{ nvm.user }}"
+  become: "{{ nvm.user }}"
   lineinfile: >
     dest=~/.profile
     line="source ~/.nvm/nvm.sh"
     create=yes
   tags: nvm
 
 - name: Install {{ nvm.node_version }}
-  command: sudo -iu {{ nvm.user }} nvm install {{ nvm.node_version }}
+  shell: source ~/.profile && nvm install {{ nvm.node_version }}
```

Comment:
Use `.` (dot) instead of `source` for maximum compatibility:

```
shell: . ~/.profile && nvm install {{ nvm.node_version }}
```

See https://github.com/leonidas/ansible-nvm/issues/8